tests/components/temper/test_sensor.py

```diff
@@ -0,0 +1,34 @@
+"""The tests for the temper (USB temperature sensor) component."""
+from datetime import timedelta
+from unittest.mock import Mock, patch
+
+from homeassistant.setup import async_setup_component
+import homeassistant.util.dt as dt_util
+
+from tests.common import async_fire_time_changed
+
+
+async def test_temperature_readback(hass):
```

Comment:
Add type hints:

```suggestion
async def test_temperature_readback(hass: Homeassistant) -> None:
```

Reply:
I've no objection to this, but I don't believe it's a general requirement to have type hints on tests.

Reply:
True, but it seems that those have been introduced widely just recently.

I am by the way sorry for the typo in `HomeAssistant` (should be capital A) in my code suggestion.